alerts/cluster-monitoring-operator/KubeAPIDown.md

```diff
@@ -7,29 +7,32 @@ been reachable by the monitoring system for more than 15 minutes.
 
 ## Impact
 
-This is a critical alert. The Kubernetes API is not responding. The
-cluster may partially or fully non-functional.
+This is a critical alert. It indicates that the Kubernetes API is not
+responding, and the cluster might be partially or fully non-functional.
 
 ## Diagnosis
 
-Check the status of the API server targets in the Prometheus UI.
+1. Verify the status of the API server targets in Prometheus in the OpenShift
+web console.
 
-Then, confirm whether the API is also unresponsive for you:
+1. Confirm whether the API is also unresponsive:
```

Comment:
shall it be `2.` and the line 24 be `3.`? 
or the document generator will give them right numbers?
my markdown render does not count the items correctly.
<img width="255" alt="image" src="https://user-images.githubusercontent.com/6040960/201441634-e5589256-e94f-4216-873b-1e4e365fe2d1.png">